code/modules/forensics/_forensics.dm

```diff
@@ -0,0 +1,233 @@
+/**
+ * Forensics datum
+ *
+ * Placed onto atoms, and contains:
+ * * List of fingerprints on the atom
+ * * List of hidden prints (used for admins)
+ * * List of blood on the atom
+ * * List of clothing fibres on the atom
+ */
+/datum/forensics
+	/// Weakref to the parent owning this datum
+	var/datum/weakref/parent
+	/**
+	 * List of fingerprints on this atom
+	 *
+	 * Formatting:
+	 * * print = print
+	 */
+	var/list/fingerprints
+	/**
+	 * List of fingerprints on this atom
+	 *
+	 * Formatting:
+	 * * ckey = realname/gloves/ckey
+	 */
+	var/list/hiddenprints
+	/**
+	 * List of fingerprints on this atom
+	 *
+	 * Formatting:
+	 * * dna = bloodtype
+	 */
+	var/list/blood_DNA
+	/**
+	 * List of clothing fibres on this atom
+	 *
+	 * Formatting:
+	 * * fiber = fibre
```

Comment:
why are they spelled different.

Reply:
sorry my canadian fingers keep trying to correct it